src/fiatconnect/FiatDetailsScreen.test.tsx

```diff
@@ -80,7 +80,7 @@ describe('FiatDetailsScreen', () => {
     fireEvent.changeText(getByTestId('input-accountNumber'), fakeAccountNumber)
 
     const expectedBody = {
-      accountName: 'n/a',
+      accountName: 'CapitalTwo Bank ...7890',
```

Comment:
The test change you'll need if you take my other suggestion
```suggestion
      accountName: 'CapitalTwo Bank (...7890)',
```